Update sql-database-long-term-backup-retention-configure.md

sagoo33 opened the pull request:
Fix listing all LTR policies for within a server and a specific database commands.

azurecla commented:
Hi __@microsoftman__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@microsoftman : Thanks for your contribution to the Azure documentation! The author, @YuriDio, has been notified to review your proposed change.

YuriDio commented:
@microsoftman thanks for your contribution. @PRMerger please merge this change.

YuriDio commented:
@BryanLa pls merge

sagoo33 committed:
Update sql-database-long-term-backup-retention-configure.md

Fix listing all LTR policies for within a server and a specific database commands.

pixia reviewed (changes requested):
Why are we making this change? The -Current switch is required to get LTR V2 policy. Without it, it will try to get LTR V1 policy and will fail since we have blocked it. Please revert the change.

anosov1960 reviewed (changes requested):
Without -Current it will fall back to teh deprecated LTRv1 API. Pls revert.
